cygwin.rst

```diff
@@ -0,0 +1,194 @@
+
+.. code-block:: rest
+
+  $ mkdir odk
+    
+You have two options: 
+
+  - Use the native virtualenv.
+  - Use virtualenvwrapper.
+
+
+.. _native-virenv:
+
+Native virtualenv
+~~~~~~~~~~~~~~~~~~~~~
+Create a new python 3 virtual environment, "odkenv" is the name of the virtualenv, you are free to add your own.
+
+.. code-block:: rest
+
+  $ virtualenv --python=<python path/python.exe> odkenv
+ 
+After creating python3 virtualenv in the previous step, multiple files are coppied into the folder odkenv.
+
+.. code-block:: rest
+
+  $ ls odkenv
+
+The folder Scripts contains all virtualenv control as bat files.
+
+To activate the odkenv:
+
+.. code-block:: rest
+
+  $ cd odkenv
+  .
+  .
+  .
+  $ cd Scripts
+
+  $ odk/odkenv/Scripts/activate.bat
+
+
+To deactivate the odkenv:
+
+.. code-block:: rest
+
+  $ odk/odkenv/Scripts/deactivate.bat
+
+.. _virenv-wrapper:
+
+Virtual Environment Wrapper
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Dealing with Virtualenv tool might be tiresome under windows. Thus, you are advised to install virtualenvwrapper tool using the following command.
+
+.. code-block:: rest
+
+  $ pip install virtualenvwrapper-win
+
+
+Create a new virtualenv use the command:
+
+.. code-block:: rest
+
+  $ mkvirtualenv odkenv
+
+Once the odkenv is created, it is autoamtically activated, the current path in cmd will appear for example as:
+
+.. code-block:: rest
+
+  $ (odkenv) C:/odk/docs
+
+To deactivate the odkenv, write:
+
+.. code-block:: rest
+
+  $ deactivate
+
+To activate the odkenv any time:
+
+.. code-block:: rest
+
+  $ workon odkenv
+
+.. _git-glfs:
+
+Git and GLFS
+-------------------------------
+  - Install `Git for windows <https://git-scm.com/downloads>`_.
+
+Make sure that git is installed properly by typing (git) in the cmd.
+
+  - Install `GLFS <https://git-lfs.github.com/>`_.
+
+
+.. _andr-tools:
+
+Android Tools
+-------------------------------
+Android tools (Adb) by installing `Android studio <https://developer.android.com/studio/index.html/>`_
+
+
+.. _fork-clone:
+
+Fork and Clone the docs repo
+-------------------------------
+From github, fork the `odk docs <https://github.com/opendatakit/docs>`_ . this will create a copy od the docs in your account called ``origin``.
+
+Then Clone docs files into your local machine.
+
+.. code-block:: rest
+
+  $ cd odk
+  .
+  .
+  .
+  $ git clone https://github.com/your-github-username/docs.git
+
+
+.. _remote-upstream:
+
+Set the Upstream Remote
+~~~~~~~~~~~~~~~~~~~~~~~~
+.. code-block:: rest
+
+  $ git remote add upstream https://github.com/opendatakit/docs.git
+
+.. _requirs:
+
+Install the requirements
+------------------------
+First activate odkenv:
+
+.. code-block:: rest
+
+  $ workon odkenv
+
+Make sure you are inside the docs folder Then run:
+
+.. code-block:: rest
+ 
+  $ pip install -r requirements.txt
+
+To this step, you will have ODKdocs environment ready. You can start change and build.
+
+Install Notepad++ to edit source files in windows from :
```

Comment:
You might suggest an editor, but people can use whatever they want.

Reply:
I agree on this.